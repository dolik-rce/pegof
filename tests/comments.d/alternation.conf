input comments.d/alternation.peg
