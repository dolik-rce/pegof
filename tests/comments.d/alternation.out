main <-
    # This is alternation comment
    "A"
    / (
        # This is multiline
        # embedded alternation comment
        #
        # with extra space in between
        "B"
        / "C"
    )
